### PR TITLE
[incubator/kubernetes-tagger] Add Kubernetes-tagger chart

### PR DESCRIPTION
**Pull request by oxyno-zeta**
#### What this PR does / why we need it:
This Pull Request will add a new Chart for a new project called [Kubernetes-tagger](https://github.com/oxyno-zeta/kubernetes-tagger). This one actual allow to tag EBS Volumes on AWS. The target is to do the same for LoadBalancers and to support other cloud provider like GCP.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by oxyno-zeta** [comment]
Hello,
I've updated the version to the latest one and rename "deploymentAnnotations" in "annotations" to avoid troubles in the future.
Tell me if something wrong.
Thanks.

**Comment by oxyno-zeta** [comment]
Hello,
Any problem on this pull request ?
Thanks,

File: incubator/kubernetes-tagger/templates/clusterrole.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{{- if .Values.rbac.create -}}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{ template "kubernetes-tagger.fullname" . }}
+  labels:
+    app: {{ template "kubernetes-tagger.name" . }}
```
Comment (cpanato) [inline incubator/kubernetes-tagger/templates/clusterrole.yaml]: please use the new labels you can find the new labels here: https://github.com/helm/helm/blob/master/docs/chart_best_practices/labels.md
also when you use the latest helm to start the project it already creates using the new labels

File: incubator/kubernetes-tagger/templates/servicemonitor.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      release: {{ .Release.Name }}
+  endpoints:
+  - port: http
+    interval: {{ .Values.prometheus.operator.serviceMonitor.interval }}
+  {{- if .Values.prometheus.operator.serviceMonitor.scrapeTimeout }}
+    scrapeTimeout: {{ .Values.prometheus.operator.serviceMonitor.scrapeTimeout }}
+  {{- end }}
+  namespaceSelector:
+    any: true
+{{ end }}
```
Comment (cpanato) [inline incubator/kubernetes-tagger/templates/servicemonitor.yaml]: new line

**Review by cpanato — CHANGES_REQUESTED** [review]

**Comment by oxyno-zeta** [comment]
Hello @cpanato ,
Thanks for your first review.
I've committed fixes for your change request.
Tell me if there is another problem.
Thanks,

File: incubator/kubernetes-tagger/templates/clusterrole.yaml (hunk 1 above)
Comment (oxyno-zeta) [reply incubator/kubernetes-tagger/templates/clusterrole.yaml]: It is fixed now. I've started the chart with helm 2.11.0, that's why :).

File: incubator/kubernetes-tagger/templates/servicemonitor.yaml (hunk 2 above)
Comment (oxyno-zeta) [reply incubator/kubernetes-tagger/templates/servicemonitor.yaml]: It is fixed.

**Comment by cpanato** [comment]
@oxyno-zeta thanks for this, can you please move this to the stable folder? then we run the tests and merge it :)

**Comment by oxyno-zeta** [comment]
Hello @cpanato !
I put the chart in the stable folder as you wish.
Tell me if you need something else :)

**Comment by oxyno-zeta** [comment]
Hello,

I've updated the chart to add support of services watch (new feature in the main project).
It is possible to merge it ?

Thanks,

**Comment by maorfr** [comment]
@cpanato do we want to push this on?

**Comment by oxyno-zeta** [comment]
Hello, 
Any news about this ?
Thanks !

**Comment by oxyno-zeta** [comment]
Hello,
Any news on this ?
Thanks !

**Comment by oxyno-zeta** [comment]
Seems that this will not be merged. Closing because need to remove my fork. Tell me if you want it.

**Pull request closed without merging**